pantheon-bundle/frontend/src/app/versions.test.tsx

```diff
@@ -210,7 +210,7 @@ describe("Versions tests", () => {
         const instance = wrapper.instance()
         wrapper.setState({ "login": true })
         wrapper.setState({
-            "results": [[{ "type": "draft", "icon": "BlankImage", "path": "/modules/test", "version": "Version 1", "publishedState": "Not published", "updatedDate": "", "firstButtonType": "primary", "secondButtonType": "secondary", "firstButtonText": "Publish", "secondButtonText": "Preview", "isDropdownOpen": false, "isArchiveDropDownOpen": false, "metadata": "" }]],
```

Comment:
should this be changed ?,here were we testing , if states get changed on publich event from draft to release ?

Reply:
changePublishState handles:
  * change draft to released
  or
  * change released to draft

On line 104 there is a test for changePublishState that matches "Publish" button. That's why I changed this test to cover the other scenario,

Reply:
I updated the tests so that it covers the click for both scenarios!